includes/controllers/class.llms.controller.orders.php

```diff
@@ -169,14 +170,44 @@ public function complete_order( $order, $old_status ) {
 
 		unset( LLMS()->session->llms_coupon );
 
-		// Trigger order complete action.
+		/**
+		 * Action fired on order complete
+		 *
+		 * Prior to the students being enrolled.
+		 *
+		 * @since Unknown
```

Comment:
This is almost certainly from version 1

```suggestion
		 * @since 1.0.0
```